tests/console/apache.pm

```diff
@@ -46,6 +46,20 @@ sub run {
     systemctl 'start apache2';
     systemctl 'status apache2';
 
+    # Check if apache is working when php module is enabled
+    if (is_sle('<=12-SP5')) {
+        zypper_call('in apache2-mod_php72 php72-curl');
+    }
+    assert_script_run('a2enmod php7');
+    systemctl 'stop apache2';
+    systemctl 'start apache2';
+    systemctl 'reload apache2';
```

Comment:
Why there is `stop` `start` `reload`? I'd keep only reload or just use `restart` ;)

Reply:
Well, I observed that the segmentation fault was happening either with `systemctl stop` either with `systemctl reload`, that's why I decided to keep it like this ^^ As for the `restart`, the regression is not caught (no segfault happening). It could be of course added though as a supplementary check :)

Reply:
Anna, I would recommend writing some brief explanatory code comments on the lines you do stop and reload, just for clarity.